adapter-base-quarkus/src/main/resources/META-INF/native-image/org.eclipse.hono/hono-adapter-base-quarkus/reflection-config.json

```diff
@@ -13,5 +13,14 @@
     "name" : "org.eclipse.hono.adapter.resourcelimits.DataVolumeAsyncCacheLoader",
     "allDeclaredMethods" : true,
     "allPublicMethods" : true
+  },
+  {
+    "name" : "org.eclipse.hono.config.FileFormat",
+    "allDeclaredConstructors" : true,
+    "allPublicConstructors" : true,
+    "allDeclaredMethods" : true,
+    "allPublicMethods" : true,
+    "allDeclaredFields" : true,
+    "allPublicFields" : true
```

Comment:
it shouldn't be necessary to add this here again as the hono-adapter-base-quarkus module depends on hono-service-base-quarkus